README.md

```diff
@@ -55,6 +56,9 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
   * https://auth0.com/ - hosted free for development SSO
   * https://getclef.com/ - new take on auth unlimited free tier for anyone not using premium features
   * https://ringcaptcha.com/ - tools to use phone number as id, available for free
+  * https://www.ssllabs.com/ssltest/ - very deep analysis of the configuration of any SSL web server
```

Comment:
I am not sure this one fits, I might remove it when I make the web app.  The list is for commercial offerings that have special cases for devops types